tests/src/test/330/scala/com/nvidia/spark/rapids/GpuIntervalUtilsTest.scala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.nvidia.spark.rapids
+
+import ai.rapids.cudf.ColumnVector
+import com.nvidia.spark.rapids.shims.GpuIntervalUtils
+import org.scalatest.FunSuite
+
+import org.apache.spark.sql.catalyst.util.DateTimeConstants.{MICROS_PER_DAY, MICROS_PER_SECOND}
+import org.apache.spark.sql.types.{DayTimeIntervalType => DT}
+
+class GpuIntervalUtilsTest extends FunSuite {
+
+  test("testToDayTimeIntervalString 1") {
```

Comment:
It would be good to have a comment here about why these tests are needed separately from the PySpark tests.  In the other interval PRs it was because years weren't supported, but I'm not seeing years used here and it's not clear to me what these are testing that the PySpark tests aren't already covering or why they can't be done as PySpark tests.

Reply:
`GpuIntervalUtilsTest` is a Unit Test class for code coverage purpose. It only tested a single method `GpuIntervalUtils.toDayTimeIntervalString`, let's keep it.
The PySpark already have the IT cases in `cast_test.py`